### DISCUSSION TITLE
Department wise user

### DISCUSSION THREAD

**Discussion by jasonsan9364** [LiveHelperChat/livehelperchat#2093] [Q&A]
![module access](https://github.com/LiveHelperChat/livehelperchat/assets/156453430/4d96f8fb-025a-458a-b9cb-ac5b7bbe7829)

Need help with the above notification. I created the following:

1. Department
2. Group
3. Users

but when testing, this is the message I came across. Any help would be great

**Comment by remdex** [comment]
make sure your operator has that permission... There is nothing to help just missing permissions...

**Reply by jasonsan9364** [reply]
thanks for the quick response.
